### DISCUSSION TITLE
Weapon (bow) invisible after save load, Skyrim SE

### DISCUSSION THREAD

**Discussion by Sedobobyi** [BadDogSkyrim/PyNifly#253] [Q&A]
Hello, after importing Nif (bow) into the Blender and not editing it, then immediately export it and load into the game, everything works great, except if it was equipped in the last save, after reloading that save, it appears invisible in hands, but after reequipping, it works as expected. 

What should I do to fix this?

**Comment by BadDogSkyrim** [comment]
Which bow? Which game? 

I have no idea what might cause this. Probly there's some flag that's not getting set right. Maybe.

**Reply by Sedobobyi** [reply]
I tested it with a wooden bow, Skyrim SE. I think it has something to do with the bones import and export

**Reply by Sedobobyi** [reply]
Anyway, thank you for your addon, an excellent job was done!
